debian/control

```diff
@@ -0,0 +1,11 @@
+Source: ocf-utils
+Section: python
+Priority: extra
+Maintainer: Open Computing Facility <help@ocf.berkeley.edu>
+Build-Depends: debhelper (>= 9), python, dh-virtualenv (>= 0.8)
+Standards-Version: 3.9.6
+
+Package: ocf-utils
+Architecture: any
+Depends: ${python:Depends}, ${misc:Depends}, ${shlibs:Depends}, virtualenv | python-virtualenv
```

Comment:
Does this account for both Python 2 and Python 3?

Reply:
I'm afraid I'm not sure what you mean. Account for 2 and 3 in what way?

Reply:
It looks to me like due to the shebangs that python2 is used from the system whereas python3 is used from inside the packaged venv, which is fine. Definitely makes the packaging a bit easier anyway since we only have 2 python2 scripts and want to remove them soon.

Reply:
But this package will include a dependency on both `python` and `python3` until such time as the last Python 2 script is migrated, no?

Reply:
I guess it needs `python3.7` more specifically, in addition to `python`.

Reply:
Well, the Python 2 requirement should be able to be removed now that the last two scripts have been migrated to Python 3.